src/pages/ScreenRouter/ScreenMap.ts

```diff
@@ -0,0 +1,40 @@
+import type { ScreenComponent } from '#types/screens';
```

Comment:
Small suggestion to rename the ScreenRouter folder to ScreenRouting so there isn't as much confusion that there's more nested under it besides ScreenRouter.tsx.

Reply:
Done, I've also squashed all commits